docs/TROUBLE.md

```diff
@@ -29,7 +29,8 @@ Carefully install your Tx module into your EdgeTX or OpenTX radio.  Some Tx modu
 - If you use EdgeTx, check that the firmware version is 2.9.x or later.
 - Inspect the Tx module connector and ensure your Tx module is fully seated in the bay.
 - Check that the external CRSF baud rate is set to 400k; this is the only speed supported by mLRS.
-- If your Tx module is the older version of the [R9M](FRSKY_R9.md#r9m-versions), don't forget to do the Inverter Mod.
+- If you have an older FrSky radio such as the Horus X10S or X12, or the Taranis QX7, or possibly the X9D, you may need to perform an inverter mod on your radio and/or enable one bit mode in EdgeTx to provide reliable operation at 400k baud.
```

Comment:
Would suggest a shorter phrasing, e.g.

'If you have an older FrSky radio such as the Horus X10/X10S/X12S, Taranis QX7, or Taranis X9D then you will need to do the inverter mod and/or enable one bit mode in EdgeTx to provide reliable operation at 400k baud.'

Reply:
I've read indications that the X9D is probably OK at 400K (just not any faster), but I can't confirm that myself and it's probably close enough to the edge that it might need one bit mode.  I'll try to make it shorter...

Reply:
Looks good, merging.